pocs/filterwheel/filterwheel.py

```diff
@@ -176,26 +183,31 @@ def move_to(self, position, blocking=False):
             'g_04'
         """
         assert self.is_connected, self.logger.error("Filter wheel must be connected to move")
+
         if self.camera and self.camera.is_exposing:
             msg = f'Attempt to move filter wheel {self} while camera is exposing, ignoring.'
             raise error.PanError(msg)
 
+        if self.is_moving:
+            msg = f'Attempt to move filter wheel {self} while already moving. Waiting.'
+            self.logger.warning(msg)
+            self._move_event.wait()
```

Comment:
I think a default timeout is a good idea for any hardware movement, even if really long. In situations where the hardware gets disconnected while in action then these loops can go forever.

Reply:
I'm inclined to change this so that is raises an error instead of waiting. Previously made exactly the same change with camera exposures.